src/CSharp/EasyMicroservices.StorageMicroservice.Domain/Contracts/Requests/FolderRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EasyMicroservices.StorageMicroservice.Contracts
+{
+    public class AddFolderDom
+    {
+        public AddFolderDom()
+        {
+            Name = string.Empty;
+            Path = string.Empty;
+        }
+
+        [StringLength(100, MinimumLength = 5, ErrorMessage = "Folder name must be between 6 and 100 characters.")]
+        public string Name { get; set; }
+
+        [StringLength(63, MinimumLength = 5, ErrorMessage = "Folder path must be between 6 and 100 characters.")]
+        public string Path { get; set; }
+    }
+
+    public class UpdateFolderDom
```

Comment:
Don't use the 'Dom' suffix, only Contract is good.
move it to requests.
rename it to AddFolderRequestContract